supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -285,3 +285,8 @@ def __init__(self, error: str) -> None:
 
     def to_json(self) -> Dict[str, Any]:
         return {"status": self.status, "error": self.error}
+
+
+class SignOutOK(APIResponse):
+    def to_json(self):
+        return {"status": "OK"}
```

Comment:
Assign a `status = ok` var like this:
```
class UserMetadataPutAPIResponse(APIResponse):
    status: str = "OK"
    def to_json(self) -> Dict[str, Any]:
        return {"status": self.status}
```

This allows the user to directly use `res.status == "OK"` instead of `isinstance(res, SignOutOK)` which is not idiomatic.